src/Processors/QtiV2/Out/QuestionTypes/McqMapper.php

```diff
@@ -72,7 +97,11 @@ public function convert(BaseQuestionType $questionType, $interactionIdentifier,
         }
 
         $builder = new McqValidationBuilder($question->get_multiple_responses(), $valueIdentifierMap);
-        list($responseDeclaration, $responseProcessing) = $builder->buildValidation($interactionIdentifier, $question->get_validation());
+        if (isset($feedbackOptions) && !empty($feedbackOptions)) {
+            list($responseDeclaration, $responseProcessing) = $builder->buildValidation($interactionIdentifier, $question->get_validation(), $feedbackOptions);
```

Comment:
This feels a bit like unnecessary duplication - if we're always going to call the same method and the only difference is what is passed to `$feedbackOptions`, could you just always initialize `$feedbackOptions` to `[]` and remove the `else` branch below?

Looks like you already initialize it at the top of this `if`: (https://github.com/Learnosity/learnosity-qti/pull/34/files#diff-b83c6121e7dd5ee24648d67bf70bcf88R39)[https://github.com/Learnosity/learnosity-qti/pull/34/files#diff-b83c6121e7dd5ee24648d67bf70bcf88R39]. I think all you need to do is move it out of the `if`.

Reply:
Done